src/main/java/com/syncstream_dev/syncstream/model/session/Session.java

```diff
@@ -0,0 +1,45 @@
+package com.syncstream_dev.syncstream.model.session;
+
+import java.util.ArrayList;
+
+import com.syncstream_dev.syncstream.model.ChatStream;
+import com.syncstream_dev.syncstream.model.Genre;
+import com.syncstream_dev.syncstream.model.VideoInfo;
+import com.syncstream_dev.syncstream.model.user.User;
+
+public abstract class Session {
+    protected String sid;       // Session id
+    protected Genre genre;
+    protected ArrayList<User> participants;
+    protected ChatStream chatstream;
+    protected VideoInfo videoinfo;
+
+    // Create new session
+    public Session(String sid, Genre genre, ChatStream chatstream, VideoInfo videoinfo) {
```

Comment:
Variable convention: `camelCase`.

Reply:
561587fb17e4deed9055580433d3e6b8ba731c35 addresses this